src/test/kotlin/com/github/imflog/schema/registry/compatibility/CompatibilityTaskTest.kt

```diff
@@ -137,7 +137,7 @@ class CompatibilityTaskTest {
         """.trimIndent()
         coreAvsc.writeText(coreSchema)
 
-        val depAvsc = folderRule.newFile("avro/dependency_test.avsc")
+        val depAvsc = folderRule.newFile("avro/dependency.avsc")
```

Comment:
Keep the "dependency_test.avsc" I think.

Reply:
The test expect dependent file  `avro/dependency.avsc`, so with `dependency_test.avsc` test fails with error `java.io.FileNotFoundException`